### PR TITLE
Srini/work with latest kernel

### PR DESCRIPTION
**Pull request by sv641**
fixes to work with latest kernel

### CODE REVIEW

File: kkm/kkm_kontext.c [hunk 1]
```diff
@@ -411,6 +411,8 @@ int kkm_kontext_switch_kernel(struct kkm_kontext *kkm_kontext)
 	 */
 	local_irq_enable();
 
+	put_cpu();
```
Comment (paulpopelka) [inline kkm/kkm_kontext.c:414]: After this point the kontext is preemptable.
Have we done something here to ensure the rest of the code (below) is still running on the same cpu?

File: kkm/kkm_guest_entry.S [hunk 2]
```diff
@@ -224,7 +224,7 @@ kkm_switch_to_gp_asm:
 	pushq	OFF_RAX(%rdi)	# user %rax
```
Comment (paulpopelka) [inline kkm/kkm_guest_entry.S:224]: Is the change in indentation here significant?

**Review by paulpopelka — APPROVED** [review]
No issues.
Just some questions to help with future understanding.

File: kkm/kkm_kontext.c (hunk 1 above)
Comment (sv641) [reply kkm/kkm_kontext.c:414]: once state is saved there is no reason to be on the same cpu. 
Line 340 makes sure the correct tlb entries exist.

File: kkm/kkm_guest_entry.S (hunk 2 above)
Comment (sv641) [reply kkm/kkm_guest_entry.S:224]: I don't see any problem with this.
the only rule is one instruction per line and white space it could be anything.

**Pull request merged**